qt/scientific_interfaces/Indirect/IndirectLoadILL.cpp

```diff
@@ -124,12 +186,25 @@ void IndirectLoadILL::run() {
                "'"
                ",'" +
                plot + "'," + save + ")";
+    runPythonScript(pyInput);
+#else
+    emit showMessageBox("IN16B is currently the only instrument supported in "
+                        "LoadILL on Mantid Workbench.");
+#endif
   }
-  runPythonScript(pyInput);
 
   setRunIsRunning(false);
 }
 
+void IndirectLoadILL::loadILLData(std::string const &filename,
```

Comment:
Could this function go in the anonymous namespace alongside the other none-algorithm object changing functions?